src/developer/web-api/route.md

```diff
@@ -44,6 +32,8 @@ If you want DHIS2 to _POST_ the request to the route target, use the _POST_ HTTP
 POST /api/routes/postman/run
 ```
 
+For performance reasons, the maximum allowed size for a route response body is 256 kilobytes. A response body from the origin exceeding this limit will lead to the DHIS2 server returning a bad gateway error to the client.
```

Comment:
I think maybe we should also mention here the content-types that are (and are not) supported so far with the route API